### PR TITLE
 Remove AtLeastOne contributor constraint

### LINKED ISSUE #1347: Remove AtLeastOne contributor constraint
We have a constraint that every evaluation must have at least one contributor. This was useful to define at least one person with access to the results.
Now, evaluations belong to courses which must have at least one responsible. This responsible can access the results also when not explicitly being added as a contributor. When creating exam evaluations we don't want to add additional questions and use contributions only for visibility permissions. If only the responsible is involved, we currently unnecessarily have to add the responsible as a contributor. So the constraint should be removed.

### PR DESCRIPTION
**Pull request by felixrindt**
fixes #1347

### CODE REVIEW

File: evap/staff/views.py [hunk 1]
```diff
@@ -1458,8 +1460,6 @@ def user_import(request):
         success_messages=success_messages, warnings=dict(warnings), errors=errors, test_passed=test_passed))
 
 
-@manager_required
-def user_edit(request, user_id):
     @receiver(RewardPointGranting.granted_by_removal)
```
Comment (He3lixxx) [inline evap/staff/views.py]: pls outdent

Comment (felixrindt) [reply evap/staff/views.py]: resolved

File: evap/staff/views.py [hunk 2]
```diff
@@ -793,19 +793,21 @@ def evaluation_edit(request, semester_id, evaluation_id):
     return helper_evaluation_edit(request, semester, evaluation)
 
 
+
+@receiver(RewardPointGranting.granted_by_removal)
+def notify_reward_points(grantings, **_kwargs):
+    for granting in grantings:
+        messages.info(request,
```
Comment (felixrindt) [inline evap/staff/views.py]: ah we have no request here

**Comment by He3lixxx** [comment]
Code is fine for me

@janno42 do we want to have a test for creating an evaluation without a contributor.

**Comment by felixrindt** [comment]
there's a test now

**Pull request merged**